lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -9,140 +9,143 @@
 
 %>
 
+<section class="wrapper confirm-enrollment shopping-cart cart-view">
+  <%block name="custom_content">
+
+  <div class="container">
+    % if shoppingcart_items:
+      <%block name="billing_details_highlight">
+        % if order.order_type == 'business':
+          <li class="billing">${_('Billing Details')}</li>
+        % else:
+          <li class="billing hidden">${_('Billing Details')}</li>
+        % endif
+      </%block>
+
+      % if is_course_enrollment_closed:
+        <p id="expiry-msg">${_('{course_names} has been removed because the enrollment period has closed.').format(course_names=appended_expired_course_names)}</p>
```

Comment:
If `course_names` is a list of courses, then there will be one or more of them, right? If so then you must use `ngettext`.